portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,15 @@ function addRandomFact() {
   randomFact.classList.add('visible');
 }
 
-async function getMessageFromServer() {
+async function addComments() {
```

Comment:
This method is still getting rather than adding, right? I don't see a js function that does a POST to /data right now, which I think is the request format for adding a comment.

Reply:
Yes this function gets the comments. The submit button on index.html sends a POST request to /data.

Reply:
I think it should be called `getComments` or `fetchComments` for clarity's sake. While it is adding comments to the page, it's not adding comments to the datastore, so that kept confusing me.